derive_builder/src/options/mod.rs

```diff
@@ -228,7 +234,7 @@ impl<Mode> OptionsBuilder<Mode> where
                 self.setter_public(false)
             },
             "setter" => {
-                // setter implicitly enabled
+        // setter implicitly enabled
```

Comment:
hm, that indentation looks strange. It's not aligned with the next line anymore.